### PR TITLE
Fixes forced two-handed weapons being unstorable

### LINKED ISSUE #29002: Attempting to put a proto-kinetic crusher in a bag of holding doesn't work.
Trying to do so results in the crusher being dropped at your feet, even though the message "You put the proto-kinetic crusher into the bag of holding. " appears.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
fix: The kinetic crusher and other forced two-handed objects can now correctly be stored in a bag of holding.
/:cl:

Because storage code makes the user drop the item before it's moved to its new location, there was a finnicky order of things; the user would technically store the item, but then the object's `unwield()` would call and drop the item to the ground again, therefore removing it from storage.

This fixes that, by removing the item drop from `unwield()`. Dropping the item normally now makes the user unwield it; the two are no longer strictly interconnected, meaning that they can be stored normally.

Fixes #29002.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
Oh boy, time to two shot someone by pulling out a trophy enhanced crusher out of my BOH and killing them with a backstab in two hits.

**Comment by Xhuis** [comment]
You still need a bag of holding to even carry a crusher on your person. This was a bug, it is not an addition.

**Comment by Ergovisavi** [comment]
You have no idea how the crusher even works. It can't even be used on humans.

Stop shitposting.

**Comment by ChangelingRain** [comment]
Wow, you can SUUUUPER do that to.... oh.... mobs. That have more health and are highly accurate.

**Comment by Repukan** [comment]
Fwoosh you stupid fuck the crusher only works in large mobs

**Comment by Xhuis** [comment]
Good grief, give him a break. He has a right to not know about something.

**Comment by DoctoreMad** [comment]
NEWCOMERS GET OFF MY GITHUB **REEEEEEE**

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -177,16 +177,21 @@
 	else
 		unwield(user)
 
+/obj/item/weapon/twohanded/required/dropped(mob/living/user)
```
Comment (Cyberboss) [inline code/game/objects/items/weapons/twohanded.dm]: `, show_message = TRUE`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Tacolizard** [comment]
can we ban fwoosh from talking

**Comment by FantasticFwoosh** [comment]
Am i not allowed to have a under-informed joke? But yeah it's still a pretty good PVP doublehanded weapon on the cheap. Chill out.

It'd work nicely during portal storms vs ops.

**Comment by fludd12** [comment]
@FantasticFwoosh It can't be used on just _any_ mob, but specifically mobs with the `large` tag set, which presently only is set for lavaland creatures and maybe xenobio stuff? Regardless, Ian is safe for the time being.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**